blackboardvc-portlet-webapp/src/main/webapp/css/portlet.css

```diff
@@ -294,3 +294,88 @@ li.ui-timepicker-selected .ui-timepicker-duration,
     font-weight: 600;
     cursor: pointer;
 }
+
```

Comment:
Please namespace your css. This styling would effect the whole portal, not just this portlet. Look at other selectors in this file, please prefix with `.blackboardVCRoot`